authentication-profile/README.md

```diff
@@ -0,0 +1,41 @@
+# Authentication Profile
+The Authentication Profile sample use the `DefaultAuthenticationChallengeHandler` class to take care of showing an authentication dialog for logging into a `Portal`. Once authenticated against the portal the app displays information about the authenticated user's profile.
+
+![Authentication Profile App](auth-profile.png) 
+
+## Features
+* AuthenticationManager
+* DefaultAuthenticationChallengeHandler
+* PortalInfo
+* PortalUser
+
+## Developer Pattern
+The `DefaultAuthenticationChallengeHandler` is the default implementation of the `AuthenticationChallengeHandler` interface to handle authentication with a portal.  This is set on the `AuthenticationManager`. To authenticate the user you create a `Portal` object, provide the essential credentials and then asynchronously load the portal as shown below:
```

Comment:
I thought about this but since it did not have to do with the sample I left out.  I added, let me know if it reads well.